tasks/build/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -o errexit
+
+git clone /.git .
+git checkout $1
```

Comment:
How about adding comments about which parameters are passed around? This could increase the readability.

Reply:
Done.